x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/body/data_quality_details/index.test.tsx

```diff
@@ -66,14 +66,16 @@ const defaultProps: Props = {
 
 describe('DataQualityDetails', () => {
   describe('when ILM phases are provided', () => {
-    beforeEach(() => {
+    beforeEach(async () => {
       jest.clearAllMocks();
 
       render(
         <TestProviders>
           <DataQualityDetails {...defaultProps} />
         </TestProviders>
       );
+
+      await waitFor(() => {});
```

Comment:
why?

Reply:
Guessing it's to make sure the DataQuality dashboard renders? Either way, do you mind adding a comment for these @andrew-goldstein ? 😄